<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RappelzRDBTool by Glandu2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="databaseTab">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuLoad_from_file">
     <property name="title">
      <string>Load from file</string>
     </property>
     <addaction name="actionLoadFromFile"/>
     <addaction name="actionLoad_from_RDB"/>
     <addaction name="actionLoad_from_CSV"/>
    </widget>
    <widget class="QMenu" name="menuSave_to_file">
     <property name="title">
      <string>Save to file</string>
     </property>
     <addaction name="actionSaveToFile"/>
     <addaction name="actionSave_to_RDB"/>
     <addaction name="actionSave_to_CSV"/>
     <addaction name="actionSave_to_SQL_file"/>
    </widget>
    <addaction name="actionNew_Tab"/>
    <addaction name="actionClose_Tab"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_DB_Structure_DLL"/>
    <addaction name="actionManage_Db_Description_files"/>
    <addaction name="separator"/>
    <addaction name="menuLoad_from_file"/>
    <addaction name="menuSave_to_file"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_from_SQLDatabase"/>
    <addaction name="actionSave_to_SQLDatabase"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Database"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionSQL_Options"/>
    <addaction name="actionUse_hashed_files"/>
    <addaction name="actionShow_Hide_Hash_Converter"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadFromFile"/>
   <addaction name="actionSaveToFile"/>
   <addaction name="actionClose_Database"/>
   <addaction name="separator"/>
   <addaction name="actionManage_Db_Description_files"/>
   <addaction name="actionSQL_Options"/>
   <addaction name="actionUse_hashed_files"/>
  </widget>
  <action name="actionLoad_from_RDB">
   <property name="text">
    <string>Load from RDB</string>
   </property>
   <property name="statusTip">
    <string>Load a RDB file</string>
   </property>
  </action>
  <action name="actionLoad_from_CSV">
   <property name="text">
    <string>Load from CSV</string>
   </property>
   <property name="statusTip">
    <string>Load a CSV file</string>
   </property>
  </action>
  <action name="actionLoad_from_SQLDatabase">
   <property name="text">
    <string>Load from SQL database</string>
   </property>
   <property name="toolTip">
    <string>Load from SQL database</string>
   </property>
   <property name="statusTip">
    <string>Load a database from a SQL table</string>
   </property>
  </action>
  <action name="actionSave_to_RDB">
   <property name="text">
    <string>Save to RDB</string>
   </property>
   <property name="statusTip">
    <string>Save current database to a RDB file</string>
   </property>
  </action>
  <action name="actionSave_to_CSV">
   <property name="text">
    <string>Save to CSV</string>
   </property>
   <property name="statusTip">
    <string>Save the current database to a CSV file (using tab as column separator)</string>
   </property>
  </action>
  <action name="actionSave_to_SQLDatabase">
   <property name="text">
    <string>Save to SQL database</string>
   </property>
   <property name="toolTip">
    <string>Save to SQL database</string>
   </property>
   <property name="statusTip">
    <string>Save the current database to a SQL table</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit the application</string>
   </property>
  </action>
  <action name="actionSQL_Options">
   <property name="text">
    <string>SQL Options</string>
   </property>
   <property name="statusTip">
    <string>Open the SQL option dialog box</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About this application</string>
   </property>
  </action>
  <action name="actionClose_Database">
   <property name="text">
    <string>Close Database</string>
   </property>
   <property name="statusTip">
    <string>Close the current database</string>
   </property>
  </action>
  <action name="actionLoadFromFile">
   <property name="text">
    <string>Load from File</string>
   </property>
   <property name="statusTip">
    <string>Load a file, the format is guested using the type combobox or file's extension</string>
   </property>
  </action>
  <action name="actionSaveToFile">
   <property name="text">
    <string>Save to File</string>
   </property>
   <property name="statusTip">
    <string>Save the current database to a file, the format is guested from the type combobox or the file's extension</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="statusTip">
    <string>About Qt which is used by this application</string>
   </property>
  </action>
  <action name="actionNew_Tab">
   <property name="text">
    <string>New Tab</string>
   </property>
   <property name="statusTip">
    <string>Add a new tab</string>
   </property>
  </action>
  <action name="actionClose_Tab">
   <property name="text">
    <string>Close Tab</string>
   </property>
   <property name="statusTip">
    <string>Close the current tab</string>
   </property>
  </action>
  <action name="actionManage_Db_Description_files">
   <property name="text">
    <string>Manage Db Description files</string>
   </property>
   <property name="statusTip">
    <string>Manage database description files (add / remove them)</string>
   </property>
  </action>
  <action name="actionLoad_DB_Structure_DLL">
   <property name="text">
    <string>Load DB Structure DLL</string>
   </property>
   <property name="statusTip">
    <string>Load database description file(s)</string>
   </property>
  </action>
  <action name="actionUse_hashed_files">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use hashed files</string>
   </property>
   <property name="toolTip">
    <string>Use the hashed name when opening / saving RDB file as the default file name</string>
   </property>
  </action>
  <action name="actionShow_Hide_Hash_Converter">
   <property name="text">
    <string>Show/Hide Hash Converter</string>
   </property>
  </action>
  <action name="actionSave_to_SQL_file">
   <property name="text">
    <string>Save to SQL file</string>
   </property>
   <property name="toolTip">
    <string>Save to SQL script file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
